test/bc_config.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I didn't create this file

Reply:
yes, it's a beacon chain by-product.  don't check it in.  we should actually add it to .gitingore.